proxyd/rewriter_test.go

```diff
@@ -148,6 +148,16 @@ func TestRewriteRequest(t *testing.T) {
 			expected:    RewriteOverrideError,
 			expectedErr: ErrRewriteBlockOutOfRange,
 		},
+		{
```

Comment:
Recommend some additional tests

- Range from `earliest` to `latest`
- No `toBlock` defined
- No `fromBlock` defined

Reply:
good suggestion on `earliest`, picked up a bug where `rewriteTags` doesn't rewrite `earliest` or `pending`